Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView_Tests.swift

```diff
@@ -112,6 +112,27 @@ class ChatChannelListItemView_Tests: XCTestCase {
         view.content = content
         AssertSnapshot(view)
     }
+
+    func test_ItemView_usesCorrectUIConfigTypes_whenCustomTypesDefined() {
+        // Create default ChatChannelListVC which has everything default from `UIConfig`
+        let itemView = ChatChannelListItemView()
+
+        // Create new config to set custom types...
+        var customConfig = itemView.uiConfig
+
+        class OtherLabel: UILabel { }
+        class OtherView: UIView { }
+
+        customConfig.channelList.itemSubviews.titleLabel = OtherLabel.self
+        customConfig.channelList.itemSubviews.subtitleLabel = OtherLabel.self
+        customConfig.channelList.itemSubviews.timestampLabel = OtherLabel.self
```

Comment:
I know it's annoying but these kinds of tests are there mainly to prevent copy/paste mistakes. So ideally, every subview should be a unique type. Maybe you can prepare some boilerplate types and share them across all tests?
```
class TestView1: UIView {}
class TestView2: UIView {}
...
class TestView10: UIView {}
```

Reply:
Created `TestCustomUIKitSubclasses` struct which contains some of the boilerplate empty subclasses and replaced it :)